bigtable-hbase/src/main/resources/com/google/cloud/bigtable/hbase/bigtable-hbase.properties

```diff
@@ -0,0 +1 @@
+project.version=${project.version}
```

Comment:
the project name and jar file are called bigtable-hbase.  We shouldn't need to specify the versions of hbase, netty and grpc, but rather the versions of our bigtable-hbase.jar, since that ought to be clear by looking at the pom file that generated the bigtable-hbase.jar.

As an aside, we also ought to ship our jar without grpc and netty.  Netty has a stable version out there and grpc's stable version is coming soon.